release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1106,10 +1108,20 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 [id="ocp-4-17-node-bug-fixes_{context}"]
 ==== Node
 
+* Previously, the Container Runtime Config controller did not consider whether a mirror configuration was in use when adding the scope from a `ClusterImagePolicy` CR to /etc/containers/registries.d/sigstore-registries.yaml. As a consequence, image verification failed with a `Not looking for sigstore attachments` message. With this fix, images are pulled from the mirror registry as expected. (link:https://issues.redhat.com/browse/OCPBUGS-36344[*OCPBUGS-36344*])
+
+* Previously, a group ID was not added to the /etc/group within a container when the `spec.securityContext.runAsGroup` attribute was set in the pod spec. With this release, this issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-39478[*OCPBUGS-39478*])
```

Comment:
```suggestion
* Previously, a group ID was not added to the `/etc/group` directory within a container when the `spec.securityContext.runAsGroup` attribute was set in the pod specification. With this release, this issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-39478[*OCPBUGS-39478*])
```
Hm, do we use "spec" or "specification" in plain language? Specification is probably better, but I don't see a rule.

Reply:
:heavy_check_mark: